Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Settings/SettingsRegistry.h>
+#include <AzFramework/Viewport/ViewportControllerList.h>
+#include <AzFramework/Viewport/CameraInput.h>
+#include <Atom/RPI.Public/ViewportContext.h>
+#include <AtomToolsFramework/Viewport/ModularViewportCameraController.h>
+
+#include <EMStudio/AnimViewportWidget.h>
+#include <EMStudio/AnimViewportRenderer.h>
+#include <EMStudio/AnimViewportSettings.h>
+
+namespace EMStudio
+{
+    AnimViewportWidget::AnimViewportWidget(QWidget* parent)
+        : AtomToolsFramework::RenderViewportWidget(parent)
+    {
+        setObjectName(QString::fromUtf8("AtomViewportWidget"));
+        QSizePolicy qSize(QSizePolicy::Preferred, QSizePolicy::Preferred);
+        qSize.setHorizontalStretch(0);
+        qSize.setVerticalStretch(0);
+        qSize.setHeightForWidth(sizePolicy().hasHeightForWidth());
+        setSizePolicy(qSize);
+        setAutoFillBackground(false);
+        setStyleSheet(QString::fromUtf8(""));
+
+        m_renderer = AZStd::make_unique<AnimViewportRenderer>(GetViewportContext());
+
+        SetupCameras();
+        SetupCameraController();
+    }
+
+    void AnimViewportWidget::SetupCameras()
+    {
+        m_pivotRotateCamera = AZStd::make_shared<AzFramework::RotateCameraInput>(EMStudio::ViewportUtil::BuildRotateCameraInputId());
+
+        const auto translateCameraInputChannelIds = EMStudio::ViewportUtil::BuildTranslateCameraInputChannelIds();
+        m_pivotTranslateCamera = AZStd::make_shared<AzFramework::TranslateCameraInput>(
+            translateCameraInputChannelIds, AzFramework::LookTranslation, AzFramework::TranslatePivot);
+        m_pivotTranslateCamera.get()->m_translateSpeedFn = []
+        {
+            return 3.0f;
```

Comment:
suggestion: We might want to make this a setting too (ideally a custom Emfx one as you might want a different default speed for the EMFX camera)